test/facts/get-years-since-last-action.test.js

```diff
@@ -0,0 +1,77 @@
+const moment = require('moment')
+const getYearsSinceLastActionFact = require('../../facts/get-years-since-last-action')
+
+describe('years since last action fact', () => {
+  test('should return years when there is a matching previous actions for FG1', async () => {
```

Comment:
typo in test description: "a matching previous actions" => "a matching previous action" (remove the last "s")